nanoFramework.Device.Bluetooth/SPP/IBluetoothSpp.cs

```diff
@@ -0,0 +1,71 @@
+﻿//
```

Comment:
why this SPP element should be checked in there?
Would that be better to have a separated nuget like any other IoT.Device binding? I would better see implementation of specific binging in the IoT.Device repository.

Reply:
I did have this originally but thought that it would be a very popular use case that we could include in main assembly
I can remove again and make this separate then we can have other SPP implementations as well. Ericsson etc

Reply:
The sample i have uses the SPP

Reply:
If it's specific I would really separate it. That can seat in the IoT.Device repo. It's "easy" to just add the nuget. and if the serial is specific to various vendors, then, one more reason to have it outside of generic implementation.

Reply:
I have removed from project, just need to find a place to put it under IoT.Device
Not really a device as such these Bluetooth Profiles.
Maybe have a section under devices for Bluetooth

So various bluetooth projects under
nanoFramework.IoT.Device/devices/bluetooth

@Ellerbach - Does that work ?

Reply:
This SPP it's BT generic it has nothing vendor specific.
Any specifics would be handled at native code at HAP/PAL level (just like UART and such).
I'm not seeing this being reused anywhere, so I would argue that there is no value on extracting this to another nuget package.

Reply:
ok, so if it's generic, if it's working everywhere, I'm fine keeping it. Anything vendor specific should not be here. Just asking to have the right choice done :-)

Reply:
Low Energy Bluetooth there is no standard SPP. A few companies have done there own and now become the defacto standard. What i implemented was the Nordic SPP version which is used in most places for SPP and is supported by mobile bluetooth terminal apps.
With this you can just connect with  Bluetooth LE Terminal program and send text messages to nanoFramework. Could easily be used for settig options like WiFi credentials. On Android i have been using the "Serial Bluetooth Terminal" for testing.

So I'll put it back in ?
@Ellerbach @josesimoes

Reply:
I stand corrected. The SPP is indeed for standard BT, not the LE. Because of the BLE specs this has to be provided as a custom service. 
Having said that and considering that, at this point, this feature it's available only for ESP32, I vote for having this here (as it was). If and when other BLE implementation are added, we can check what's there equivalent to this (if any) and adjust, if needed.

Reply:
@AdrianSoundy keep it here then. I understand the points now. So let's have it in this nuget!

Reply:
Added back